Specs/Scene/ModelSpec.js

```diff
@@ -1581,6 +1590,88 @@ defineSuite([
         });
     });
 
+    it('loads a glTF with WEB3D_quantized_attributes POSITION and NORMAL', function() {
+        return loadModel(boxQuantizedUrl).then(function(m) {
+            verifyRender(m);
+            primitives.remove(m);
+        });
+    });
+
+    it('loads a glTF with WEB3D_quantized_attributes POSITION and NORMAL where primitives with different accessors use the same shader', function() {
+        return loadModel(milkTruckQuantizedUrl).then(function(m) {
+            verifyRender(m);
+            primitives.remove(m);
+        });
+    });
+
+    it('loads a glTF with WEB3D_quantized_attributes POSITION and NORMAL where one primitive is quantized and the other is not, but both use the same shader', function() {
+        return loadModel(milkTruckQuantizedMismatchUrl).then(function(m) {
+            verifyRender(m);
+            primitives.remove(m);
+        });
+    });
+
+    it('loads a glTF with WEB3D_quantized_attributes TEXCOORD', function() {
+        return loadModel(duckQuantizedUrl).then(function(m) {
+            verifyRender(m);
+            primitives.remove(m);
+        });
+    });
+
+    it('loads a glTF with WEB3D_quantized_attributes JOINT and WEIGHT', function() {
+        return loadModel(riggedSimpleQuantizedUrl).then(function(m) {
+            verifyRender(m);
+            primitives.remove(m);
+        });
+    });
+
+    function testBoxSideColors(m) {
+        var rotateX = Matrix3.fromRotationX(CesiumMath.toRadians(90.0));
+        var rotateY = Matrix3.fromRotationY(CesiumMath.toRadians(90.0));
+        var rotateZ = Matrix3.fromRotationZ(CesiumMath.toRadians(90.0));
+
+        // Each side of the cube should be a different color
+        var oldPixelColor = scene.renderForSpecs();
+        for(var i = 0; i < 6; i++) {
+            expect(oldPixelColor).not.toEqual([0, 0, 0, 255]);
```

Comment:
It might be cleaner to move this above and out of the loop (and below with the other `expect` if needed).  As is, it is a bit hard to follow.